ttnn/cpp/ttnn/operations/moreh/moreh_sgd/moreh_sgd.cpp

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_sgd.hpp"
+
+#include "ttnn/operations/moreh/moreh_sgd/device/moreh_sgd_device_operation.hpp"
+
+namespace ttnn::operations::moreh::moreh_sgd {
+std::vector<std::optional<Tensor>> MorehSgd::invoke(
+    const Tensor& param_in,
+    const Tensor& grad,
+    const std::optional<const Tensor> momentum_buffer_in,
```

Comment:
Use const std::optional<Tensor>&

Reply:
resolved in [a981edd](https://github.com/tenstorrent/tt-metal/pull/12378/commits/a981edd24c59337a422bc4c59ae88b14489c75a5)